README.md

```diff
@@ -2,26 +2,24 @@
+  - You can use the infrastructure template with either of the application templates, or with any other application that satisfies the infra template’s application requirements.
+You can also use the infrastructure template with multiple applications at a time, but it will require a few steps to accomplish that configuration (see What’s not available out of the box (yet))
+- [TypeScript Nextjs application template](https://github.com/navapbc/template-application-nextjs) - Template for user-facing web applications built with Next.js.
+  - This template can be used to build traditional server-based web applications. It can also be used to build APIs. However, if your project has a significant data engineering (e.g. ETL jobs and other batch processing jobs) component, a Python-based tech stack is recommended (see the Python Flask template).
+  - This template can also be used to build static sites, but is currently not configured that way out of the box and will require some effort (see What’s not available out of the box (yet))
+- [Python Flask application template](template-application-flask](https://github.com/navapbc/template-application-flask) - Template for APIs and offline processes built with Python Flask.
+  - This template can be used to build APIs. It can also be used to build simple user-facing applications. However, if your  application is heavily on the frontend, you may want to consider using the Nextjs template since that comes already configured with internationalization, the US Web Design System, and Storybook.
 
 ### Getting started
 
-A project repository can be created using one of the application templates above. This can be done one of two ways:
-
-1. [Create a new repo](https://github.com/new) and select the desired template.
-1. From the template repo itself: Click "Use this template".
-
-Once your project repo is created:
+To get started using the templates, first create a git repo for your project.
 
-1. Copy the [pull request template](./.github/pull_request_template.md). [Learn more about PR templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository).
-1. [Configure your repo settings](./docs/setup-new-repo.md)
+1. If you are starting from scratch and need infrastructure, then start with the [infra template](https://github.com/navapbc/template-infra) and follow the instructions on the template's README to install the template in a single command line.
+2. Choose which application template you want to use based on whether you are building a primarily user-facing application (use the [Nextjs template](https://github.com/navapbc/template-application-nextjs)) or primarily backend system (use the [Flask template](https://github.com/navapbc/template-application-flask)), and follow the instructions on the respective template's README to install the template.
```

Comment:
It could be a nice touch to add how they can ask for help / get support, if they can do so

Reply:
oh hmm, that's a good question. thanks for raising it